apps/api/start/routes.ts

```diff
@@ -68,3 +68,29 @@ Route.resource('users', 'UsersController')
 Route.post('/register', 'AuthController.register')
 Route.post('/login', 'AuthController.login')
 Route.post('/logout', 'AuthController.logout').middleware('auth:api')
+
+// Routes for creating, updating, and deleting help requests
+Route.post('help-requests', 'HelpRequestController.store')
```

Comment:
These might be redundant, but I'll add them for now.

Reply:
there will be duplicate routes between controllers right
i just saw that
i will do it dynamically 
eg:
Route.put('/:controller/:id', ({ params }) => {
  return `${params.controller}Controller.update` 
})